subprojects/scala/src/main/java/org/gradle/api/tasks/scala/ScalaDocOptions.java

```diff
@@ -161,15 +163,22 @@ public void setBottom(String bottom) {
 
     /**
      * Returns the style sheet to override default style.
+     *
+     * @deprecated Scaladoc does not support to set a stylesheet any more (Scala 2.11).
      */
+    @PathSensitive(PathSensitivity.ABSOLUTE)
     @InputFile @Optional
+    @Deprecated
     public File getStyleSheet() {
         return styleSheet;
     }
 
     /**
      * Sets the style sheet to override default style.
+     *
+     * @deprecated Scaladoc does not support to set a stylesheet any more (Scala 2.11).
      */
+    @Deprecated
     public void setStyleSheet(File styleSheet) {
```

Comment:
Do we have an integration test for this? Can they use an older version of the tool that still supports this parameter?